Header header
float64 cadence
float64 cadence_avg
float64 dst #double support time
leg_params leg1
leg_params leg2
